### PR TITLE
ci: release fail notifications

### PR DESCRIPTION
**Pull request by mshanemc**
### What does this PR do?
failures for various cli release/promote workflows

### CODE REVIEW

File: .github/workflows/releaseFail.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# test
```
Comment (iowillhoit) [inline .github/workflows/releaseFail.yml:1]: 👀

File: .github/workflows/releaseFail.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        with:
+          payload: |
+            {
+              "text": "Release failed: ${{ github.event.workflow_run.repository.name }}",
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": "Release failed: ${{ github.event.workflow_run.repository.name }}"
```
Comment (iowillhoit) [inline .github/workflows/releaseFail.yml:33]: Let's wrap this line with `:attentioncse:` (or something similar) so it stands out

File: .github/workflows/releaseFail.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      - name: Announce Failure
+        id: slack
+        uses: slackapi/slack-github-action@v1.21.0
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.CLI_ALERTS_SLACK_WEBHOOK }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        with:
+          payload: |
+            {
+              "text": "Release failed: ${{ github.event.workflow_run.repository.name }}",
```
Comment (iowillhoit) [inline .github/workflows/releaseFail.yml:27]: I am not super familiar with Slack's Block syntax. Is this `text` that is outside of the `blocks` array correct?

File: .github/workflows/releaseFail.yml (hunk 2 above)
Comment (iowillhoit) [reply .github/workflows/releaseFail.yml:33]: Can you get the name of the workflow that failed? "Release failed" does not apply to some of the workflows we are watching

File: .github/workflows/releaseFail.yml (hunk 3 above)
Comment (mshanemc) [reply .github/workflows/releaseFail.yml:27]: it's the fallback text (ex: notificaiton on your phone where the richer stuff isn't supported)

https://api.slack.com/reference/messaging/payload

File: .github/workflows/releaseFail.yml (hunk 2 above)
Comment (mshanemc) [reply .github/workflows/releaseFail.yml:33]: see `workflow name` below.  I can put that in the title, too.

File: .github/workflows/releaseFail.yml (hunk 3 above)
Comment (iowillhoit) [reply .github/workflows/releaseFail.yml]: Good to know, thanks 👍

**Review by iowillhoit — DISMISSED** [review]

**Pull request merged**